Pre-processing/epochs.py

```diff
@@ -0,0 +1,69 @@
+import os
+import mne
+import numpy as np
```

Comment:
Also this code seems to already exists in the MNE package. Why are we writing it locally instead of calling it from the package?